src/Microsoft.AspNetCore.ResponseCaching/Internal/DistributedResponseCacheStore.cs

```diff
@@ -42,17 +60,51 @@ public DistributedResponseCacheStore(IDistributedCache cache)
             catch { }
```

Comment:
Is there any exception that we'd actually like catch for logging purposes? Or do we not care at all since this is `Remove`?